packages/grid/devspace.yaml

```diff
@@ -36,7 +36,7 @@ images:
   backend:
     image: "${CONTAINER_REGISTRY}/${DOCKER_IMAGE_BACKEND}"
     buildKit:
-      args: ["--platform", "linux/${PLATFORM}"]
+      args: ["--target", "backend", "--platform", "linux/${PLATFORM}"]
     dockerfile: ./backend/backend.dockerfile
     target: "backend"
```

Comment:
Q: We already have the target as backend here, why do we also include it in the args?

Reply:
Keeping it explicit and in-parity with cd-syft-*.yaml

```yaml

      - name: Build and push `grid-backend` image to DockerHub
        id: grid-backend-build
        uses: docker/build-push-action@v5
        with:
          context: ./packages
          file: ./packages/grid/backend/backend.dockerfile
          platforms: ${{ steps.release_metadata.outputs.release_platform }}
          target: backend # <---

```

Reply:
Ahh ok, but why do we repeat in args and as well as in the target in devspace.yaml?

Reply:
ah yes - merge conflict got pushed in

Reply:
Ah my bad - I didn't see that. will fix.